<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>261</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Choose Target Language</string>
  </property>
  <layout class="QVBoxLayout" name="Layout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Please Choose a target language</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="comboBoxLable">
          <property name="text">
           <string>Language: </string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="langComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="langExistGroupBox">
        <property name="title">
         <string>Transaltion Already Exists</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="langExistMsg">
           <property name="text">
            <string>A selected language translation already exists. What would you like to do?</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="createNewTranslation">
           <property name="text">
            <string>Create new translation anyway</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="editExistTranslation">
           <property name="text">
            <string>Edit the exist translation</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>langComboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>onLangPick(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>acceptedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>rejectedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLangPick(QString)</slot>
  <slot>acceptedSlot()</slot>
  <slot>rejectedSlot()</slot>
 </slots>
</ui>
